test/smoke_test.py

```diff
@@ -17,6 +17,7 @@ def smoke_test_torchvision() -> None:
         all(x is not None for x in [torch.ops.image.decode_png, torch.ops.torchvision.roi_align]),
     )
 
+
```

Comment:
Changes in this file are the lint fixes, right?

Reply:
Correct. I believe the offending PR is #7020